### PR TITLE
config version to v7

### PR DESCRIPTION
**Pull request by parfa30**
Updated version of config based on this pull request: https://github.com/lsst-ts/ts_config_ocs/pull/253

### CODE REVIEW

File: python/lsst/ts/ess/csc/config_schema.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 $schema: http://json-schema.org/draft-07/schema#
 $id: https://github.com/lsst-ts/ts_ess/blob/main/python/lsst/ts/ess/csc/config_schema.py
 # title must end with one or more spaces followed by the schema version, which must begin with "v"
-title: ESS v6
+title: ESS v7
```
Comment (wvreeven) [inline python/lsst/ts/ess/csc/config_schema.py:31]: Thanks for following the procedure for increasing the version number when adding new telemetry. As it happens, v6 is not in use yet at the summit so please revert this commit.

**Review by wvreeven — CHANGES_REQUESTED** [review]
See my review comment.

File: python/lsst/ts/ess/csc/config_schema.py (hunk 1 above)
Comment (parfa30) [reply python/lsst/ts/ess/csc/config_schema.py:31]: Instead, can I just cancel this Pull Request?

Comment (wvreeven) [reply python/lsst/ts/ess/csc/config_schema.py:31]: Yes, of course. Please cancel the PR and delete your branch.

**Review by wvreeven — APPROVED** [review]
v6 has hit the summit so this indeed should be updated to v7.

**Comment by wvreeven** [comment]
Please first rebase your branch on `develop` before merging.

**Pull request merged**